src/lib/airtable/schema.js

```diff
@@ -27,28 +27,21 @@ export const Columns = {
 		password: {name:`Password`, type:`text`},
 		name: {name:`Name`, type:`text`},
 		paymentIds: {name:`Payments`, type:`foreignKey-many`},
-		meterReadingAndInvoiceIds: {name:`Meter Readings and Invoices`, type:`foreignKey-many`},
+		meterReadingsAndInvoiceIds: {name:`Meter Readings and Invoices`, type:`foreignKey-many`},
 		customerUpdateIds: {name:`Customer Updates`, type:`foreignKey-many`},
 		id: {name:`ID`, type:`formula`},
 		purchaseRequestIds: {name:`Purchase Requests`, type:`foreignKey-many`},
 		purchaseRequests2Ids: {name:`Purchase Requests 2`, type:`foreignKey-many`},
 		inventoryUpdateIds: {name:`Inventory Updates`, type:`foreignKey-many`},
 	},
 	"Sites": {
-		customers: {name:`CustomerData`, type:`custom-object`},
-		financialSummaries: {name: `FinancialSummaries`, type:`custom-object`},
-		tariffPlans: {name:`TariffPlans`, type:`custom-object`},
 		name: {name:`Name`, type:`text`},
 		userIds: {name:`Users`, type:`foreignKey-many`},
 		customerIds: {name:`Customers`, type:`foreignKey-many`},
-		financialSummaryIds: {name:`Financial Summaries`, type:`foreignKey-many`},
+		financialSummarieIds: {name:`Financial Summaries`, type:`foreignKey-many`},
 		tariffPlanIds: {name:`Tariff Plans`, type:`foreignKey-many`},
 		id: {name:`ID`, type:`formula`},
 		inventoryIds: {name:`Inventory`, type:`foreignKey-many`},
-		products: {name: `Products`, type: `custom-object`},
-		inventory: {name: `SiteInventory`, type: `custom-object`},
-		purchaseRequests: {name: `PurchaseRequests`, type: `custom-object`},
-		inventoryUpdates: {name: `InventoryUpdates`, type: `custom-object`},
```

Comment:
I don't think these should've changed. Can you revert these and just keep the `meterType` diff?